### ISSUE TITLE
error after updating to 2.20.0

### ISSUE THREAD

**Issue by Spell6inder** [unclead/yii2-multiple-input#280]
After updating to 2.20.0 there was an error caused by "Json::decode" in this line
https://github.com/unclead/yii2-multiple-input/commit/3d4895cb35d45535d554bccd781e8fddfc87820b#diff-b6b9e07de46562690f577fad37ac11c2R330

Config
``` php
->widget(\unclead\multipleinput\MultipleInput::class, [
//...
    'columns' => [
//...
        [
            'type' => \kartik\widgets\Select2::class,
//...
            'options' => [
                'pluginOptions' => [
//...
                    'escapeMarkup' => new \yii\web\JsExpression('function (markup) { return markup; }'),
                    'templateResult' => new \yii\web\JsExpression('function(product) { return product.text; }'),
                    'templateSelection' => new \yii\web\JsExpression('function (product) { return product.text; }'),
                ],
            ],
        ],
    ],
]);
```
part of result json that try decoded
``` javascript
{
//...
	"pluginOptions": {
		"escapeMarkup": function(markup) {return markup;},
		"templateResult": function(product) {return product.text;},
		"templateSelection": function(product) {return product.text;}
	}
}
```
but got syntax error becose anonymous functions is not valid json
I believe that it is necessary to find an alternative way to solve #261

**Comment by Spell6inder** [comment]
maybe json encode/decode change to serrialize/unserrialize....

**Comment by unclead** [comment]
yes, we can use `serialize`

**Comment by unclead** [comment]
fixed in 2.20.1 (17c77b3b4f4f8d07b563b0e30dd8e0ffaead4b55)

**Comment by Spell6inder** [comment]
Thank you for the quick response!
